Scroller/EditableWrapper.js

```diff
@@ -289,11 +289,15 @@ const EditableWrapper = (props) => {
 	}, [editable, finalizeOrders, reset]);
 
 	const handleMouseMove = useCallback((ev) => {
-		const {centeredOffset, itemWidth, selectedItem} = mutableRef.current;
+		const {centeredOffset, itemWidth, prevToIndex, selectedItem} = mutableRef.current;
+		const {clientX} = ev;
+		const scrollContentOffset = scrollContentRef.current.scrollLeft - centeredOffset;
```

Comment:
I think this PR is related to http://clm.lge.com/issue/browse/WRO-709, and we would need to fix this part to consider RTL.
```suggestion
		const scrollContentOffset = scrollContentRef.current.scrollLeft * (rtl ? -1 : 1) - centeredOffset;
```